### DISCUSSION TITLE
Width of shape and label

### DISCUSSION THREAD

**Discussion by muteboy** [terrastruct/d2#1927] [Q&A] [answered]
I would like to be able to set the width of a shape smaller than the label, and have the label overlap the edges. Is that possible? It looks like the width is always wide enough to fit the label, no matter the width setting.

**Comment by alixander** [comment, marked as answer]
![Screenshot 2024-05-02 at 11 55 39 AM](https://github.com/terrastruct/d2/assets/3120367/493cc34c-ef35-4266-afc8-1e213dc49730)

Made a PR to support this: https://github.com/terrastruct/d2/pull/1928

Will be in next release
